enos/enos-variables.hcl

```diff
@@ -154,3 +154,15 @@ variable "docker_mirror" {
   type        = string
   default     = "docker.mirror.hashicorp.services"
 }
+
+variable "boundary_license" {
```

Comment:
I assume this is a file path to a license file? Could we specify that somewhere so it's specific (whether it's the description of the variable name)

Reply:
Actually it's the license string, I'll put in code to read it from a file for ease though!